tests/unit/client.py

```diff
@@ -319,6 +319,9 @@ def test_show_node(self):
     def test_power_cycle(self):
         assert C.node.power_cycle('node-07') is None
 
+    def test_power_cycle_force(self):
```

Comment:
Please add a test for `force=False` and `force=NotABoolean` as well, so we can be complete